src/Application.cc

```diff
@@ -433,7 +433,7 @@ bool Application::LoadPlugin(const std::string &_filename,
   // Add default folder and install folder
   std::string home;
   common::env(GZ_HOMEDIR, home);
-  systemPaths.AddPluginPaths(home + "/.ignition/gui/plugins:" +
+  systemPaths.AddPluginPaths(home + "/.gz/gui/plugins:" +
```

Comment:
Here too, we can support both directories

Reply:
https://github.com/gazebosim/gz-gui/pull/454/commits/2d40757457b56309f6cab8c2330ace404481e5ba